Makefile

```diff
@@ -329,6 +329,8 @@ $(foreach ITEM,$(PILOT_GO_BINS_SHORT),$(eval $(call pilotbuild,$(ITEM))))
 .PHONY: istioctl
 istioctl ${ISTIO_OUT}/istioctl:
 	bin/gobuild.sh ${ISTIO_OUT}/istioctl ./istioctl/cmd/istioctl
+	${ISTIO_OUT}/istioctl collateral --bash && \
+	mv istioctl.bash ${ISTIO_OUT}/tools
```

Comment:
Per discussion, the _istioctl_ command on the previous line will create the file in the current directory. You are trying to move the file from the directory where the command exists, and no the current directory.

Reply:
@ericvn thanks. I've modified this in the PR.